Added option to display error messages for each files in dashboard

lafe opened the pull request:
We have the requirement that, if multiple uploads fail, an error message for each file should be displayed in the dashboard. This pull request adds the new option `showErrorIconInFileList` to the dashboard which renders the "question mark icon" of the status bar at the same place as the icon to copy the link of the file to the clipboard (as the upload failed, no link would be available and no copy icon would be displayed, so only one icon would be visible).
By default, the new icon is disabled as to not introduce new behaviour.

arturi commented:
Agreed this is a very good improvement, and about making it the default, Question mark could be hidden with CSS if needed.

lafe commented:
I changed the behavior. We might have the need that we want to disable it in certain scenarios, so I just changed the default instead of removing the option altogether (and maybe someone else has the same need, so it is easier to just flick a switch instead of tinkering with the CSS).

arturi commented:
Thank you! I moved the CSS to the Dashboard, cause it should be in theory independent from the StatusBar, and removed the option — I think we are adding too many to the Dashboard already, and visual things that can be hidden with CSS should be hidden with CSS. Otherwise I worry we’ll end up supporting things like `showFileName`.
